.github/workflows/check_files_submitted.py

```diff
@@ -0,0 +1,108 @@
+"""
+Script to check if Pull Request surpasses the designated threshold of 20 files.
+
+We strongly recommend optimizing the changes to align with our established
+guidelines.
+Please reduce the number of files to meet this standard before submission.
+Thank you for your cooperation
+
+Methodology:
+    Accepts the Base Branch and PR Branch Head references as
+    command Line arguments.
+    Checks the number of files modified in the PR branch by utilizing
+    subprocess module to run the CLI commands using git diff.
+    Checks the number of changed files returned by the 'get_changed_files'
+    by the calculating the length of the list.
+    Exits with an appropriate error message if the number of files exceeds 20.
+
+NOTE:
+    This script complies with Python 3 coding and documentation standards,
+    including:
+        1) Pylint
+        2) Pydocstyle
+        3) Pycodestyle
+        4) Flake8
+    Run these commands from the CLI to ensure the code is compliant
+    for all your pull requests.
+"""
+
+import sys
+import subprocess
+import argparse
+
+
+def get_changed_files(base_branch, pr_branch):
+    """
+    Get the list of changed files between branches.
+
+    Args:
+        base_branch (str): Base branch name.
+        pr_branch (str): Pull request branch name.
+
+    Returns:
+        list: List of changed file names.
+    """
+    try:
+        with subprocess.Popen(
+            [
+                "git",
+                "diff",
+                "--name-only",
+                f"origin/{base_branch}...{pr_branch}",
+                "--",
+            ],
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            universal_newlines=True,
+        ) as git_diff_process:
+            git_diff_output, git_diff_error = git_diff_process.communicate()
+
+            if git_diff_process.returncode != 0:
+                print(f"Error: {git_diff_error}")
+                return []
+
+            changed_files = git_diff_output.splitlines()
+            return changed_files
+    except Exception as e:
+        print(f"Error: {e}")
```

Comment:
You have an error condition and this logic will make the script terminate with a normal exit code